modules/302-vertical-pod-autoscaler/hooks/set_maxallowed.go

```diff
@@ -66,6 +68,27 @@ type VPA struct {
 	ContainerRecommendations []autoscaler.RecommendedContainerResources
 }
 
+type DeckhousePod struct {
+	IsReady bool
+}
+
+func applyDeckhousePodFilter(obj *unstructured.Unstructured) (go_hook.FilterResult, error) {
+	var isReady bool
+
+	pod := &v1.Pod{}
+	err := sdk.FromUnstructured(obj, pod)
+	if err != nil {
+		return nil, fmt.Errorf("cannot parse pod object from unstructured: %v", err)
+	}
+	for _, cond := range pod.Status.Conditions {
+		if cond.Type == v1.PodReady && cond.Status == v1.ConditionTrue {
+			isReady = true
+			break
+		}
+	}
+	return &DeckhousePod{IsReady: isReady}, nil
```

Comment:
We can return boolean variable immediately.
We do not need wrap boolean var with struct.

Reply:
yep